.vscode/launch.json

```diff
@@ -24,6 +24,17 @@
             "program": "${file}",
             "envFile": "${workspaceRoot}/.env",
             "console": "externalTerminal",
-        }
+        },
+        {
+            "name": "Test Query (external)",
```

Comment:
Let's remove the lauch.json changes. We should have pytests that launch mssql-cli in non-interactive mode, and validate the output.

Reply:
We should definitely have pytests—however the usefulness with launch.json is purely for debugging purposes so that we can use breakpoints. Is it possible to also use pytest in this capacity?

Reply:
I see. It would be nice if we can step through the pytest hosted mssql-cli process as well, though I'm not sure what hoops we need to jump through to enable that.